### PR TITLE
Update CruiseInformation.csv

### PR DESCRIPTION
**Pull request by cdobs**
Updating cruiseInformation.csv with 2 historical cruises, both were CSPP recoveries

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -42,13 +42,15 @@ AT27,Atlantis,2015-04-28T16:00:00,2015-05-14T12:26:00,Pioneer 4
 TN-323,Thomas G. Thompson,2015-05-29T16:00:00,2015-06-13T04:00:00,Papa 3 #TO BE DELETED
 TN323,Thomas G. Thompson,2015-05-29T16:00:00,2015-06-13T04:00:00,Papa 3
 PS1506A,Pacific Storm & Elakha ,2015-06-03T00:00:00,2015-06-03T23:59:59,Endurance 3b
+CT20150612,Connecticut,2015-06-12T00:00:00,2015-06-12T23:59:59,Pioneer ISSP recovery
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:45]: looks good - verified as much as possible via the link that Collin 'slacked' to data team.

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -42,13 +42,15 @@ AT27,Atlantis,2015-04-28T16:00:00,2015-05-14T12:26:00,Pioneer 4
...
 EK-1506,Elakha,2015-06-23T00:00:00,2015-06-23T23:59:59,Endurance CE01ISSP
 MH20150702,Matthew Hughes,2015-07-02T00:00:00,2015-07-02T23:59:59,Pioneer glider cp_340 recovery
 TN326,Thomas G. Thompson,2015-07-04T00:00:00,2015-08-07T23:59:59,Cabled (rvdata)
 ML-1507,Miss Linda,2015-07-08T00:00:00,2015-07-08T23:59:59,Endurance CE06ISSP
 EK-1507,Elakha,2015-07-17T00:00:00,2015-07-17T23:59:59,Endurance CE01ISSP
 SI-1507,Scarlett Isabella,2015-07-22T00:00:00,2015-07-26T23:59:59,# TO BE DELETED
 SI20150722,Scarlett Isabella,2015-07-22T00:00:00,2015-07-26T23:59:59,Pioneer 4a
+MGL1512,Marcus G. Langseth,2015-07-30T00:00:00,2015-08-28T23:59:59,Pioneer CNSP recovery
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:53]: Looks good - verified via link to cruise fr Collin (sent via slack to data team channel).  Confirmed that ship exists on shiplist.csv

**Review by rtravis-whoi — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv:45]: Looks good.

File: cruise/CruiseInformation.csv (hunk 2 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv:53]: Looks good.

**Review by snwhite — APPROVED** [review]
Additions look good.

**Pull request merged**